contrib/orafce/sql/orafce.sql

```diff
@@ -984,3 +984,26 @@ SELECT oracle.unistr('wrong: \+2FFFFF');
 SELECT oracle.unistr('wrong: \udb99\u0061');
 SELECT oracle.unistr('wrong: \U0000db99\U00000061');
 SELECT oracle.unistr('wrong: \U002FFFFF');
+
+--
+-- test extended type in package
+--
+CREATE OR REPLACE PACKAGE ora_type AS
+FUNCTION tfunc(name VARCHAR2) RETURN NUMBER;
+END ora_type;
+/
+
```

Comment:
I think the test case should be part of the src/test/regress/sql/packages.sql with search_path set to oracle.

Reply:
I think this test case just needs to be added in orafce because this is mainly to test the data types created in orafce.And if  orafce extension is no installed,add this test case in package.sql will throws error.